src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -280,7 +280,10 @@
   <ItemGroup />
   <ItemGroup>
     <EmbeddedResource Include="Creation\CastleDynamicProxy\DynamicProxyResources.resx" />
-    <EmbeddedResource Include="ExceptionMessages.resx" />
+    <EmbeddedResource Include="ExceptionMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionMessages1.Designer.cs</LastGenOutput>
+    </EmbeddedResource>

```

Comment:
I have to say I don't understand why things would be unlinked (I rarely use resxes, so my knowledge is limited). I think, if it's not too much bother, best action for now would be to undo the changes here and fix up `ExceptionMessages.Designer.cs` (which may need manual editing at this point). Then if there's some auto-building to fix, I think it'll apply to all the resxes in the solution and we should handle it separately.